openfl-tutorials/interactive_api/Tensorflow_Word_Prediction/envoy_folder/shard_config.yaml

```diff
@@ -0,0 +1,5 @@
+# https://www.gutenberg.org/files/36668/36668-h/36668-h.htm
+template: shard_descriptor.NextWordShardDescriptor
+params:
+  title: Polish Fairy Tales
```

Comment:
Can we add all 3 shard_configs with different texts for clarity?

Reply:
Fixed as well.